File: RMSDemo/RMSDemo/Scripts/Farmer.js
$(document).ready(function () {
    $('#UserTableContainer').jtable({
        title: 'Company List',
        paging: true, //Enable paging
        pageSize: 10,
        sorting: true, //Enable sorting
        async: false,
        defaultSorting: 'ID ASC',
        actions: {
            listAction: '/Farmer/GetFarmers',
            deleteAction: '/Farmer/DeleteFarmer',
            updateAction: '/Farmer/UpdateFarmer'
        },
        fields: {
            Id: {
                title: 'ID',
                key: true,
                create: false,
                edit: false,
                list: false
            },
            FarmerName: {
                title: 'Name',
                edit: true,
            },
            Name: {
                title: 'Device'
            },
            IMEI: {
                title: 'IMEI',
                edit: false
            },
            DistributorFirstLastName: {
                title: 'Distributor',
                edit: false
            },
            FarmerCompanyName: {
                title: 'CompanyName',
                edit: true,
            },

            Pono: {
                title: 'Pono',
                list: false
            },

            DistrictId: {
                title: 'DistrictId'
            },
            PhoneNumber: {
                title: 'PhoneNumber'
            },
            StateName: {
                title: 'State'
            },
            VillageName: {
                title: 'Village'
            },
            MandalName: {
                title: 'Mandal'
            },
            Department: {
                title: 'Department'
            },
            Lattitude: {
                title: 'Lattitude'
            },
            Longitude: {
                title: 'Longitude'
            }
        }
    });
    $('#UserTableContainer').jtable('load');
    //Re-load records when user click 'load records' button.
    $('#LoadRecordsButton').click(function (e) {
        e.preventDefault();
        $('#UserTableContainer').jtable('load', {
            name: $('#name').val()
        });
    });
    //Load all records when page is first shown
})

File: RMSDemo/Entities/Farmer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities
{
    public class Farmer : RegisteredDevice
    {
        public int Id { get; set; }
        public string FarmerName { get; set; }
        public int DeviceId { get; set; }
        public string DeviceName { get; set; }
        public string Pono { get; set; }
        public string DistrictId { get; set; }
        public string PhoneNumber { get; set; }
        public int StateId { get; set; }
        public int VillageId { get; set; }
        public int MadnalId { get; set; }
        public string StateName { get; set; }
        public string VillageName { get; set; }
        public string MandalName { get; set; }
        public string Department { get; set; }
        public string Lattitude { get; set; }
        public string Longitude { get; set; }
        public int DistId { get; set; }
        public string FarmerCompanyName { get; set; }
        public string DistributorFirstLastName { get; set; }
    }
}